app/workers/daily_grouping_updater.rb

```diff
@@ -0,0 +1,27 @@
+class DailyGroupingUpdater
+  include Sidekiq::Worker
+
+  def initialize
+    @system_account = Watcher.find_by_email("kairos-robot@omadahealth.com")
+    PaperTrail.whodunnit = @system_account
+  end
+
+  def perform(app_name: nil)
+    raise "An app_name must be specified." unless app_name.present?
```

Comment:
Good point - I only did it to give a better error message to whoever was trying to run the worker but it's not necessary.